your-result page

MarcoSaba71ni opened the pull request:
### What does this PR do?
This PR contains a header, buttons, sidebar, footer, table score and responsiveness

### Description of Task to be completed?
Buttons might be completed due to not red color responsiveness
### How should this be manually tested?

### Any background context you want to provide?

MarcoSaba71ni committed:
Your commit message

MarcoSaba71ni committed:
Final Page

GustavBoe reviewed (changes requested):
Hi guys! Great work on this! I am going to give you quite a few things to do on this one, and if any of them are not within your tasks, just ignore them!

### For desktop

**Header, sidebar, search bar and footer**

- [x] Get the approved ones from your fellow students, so that we don´t code something multiple times. You can use the ones from develop branch: student-profile.html.


**Table**

- [x] Remove the long delete button at the top of the table.
- [x] Use the classes in Styles.css to change the bg colors of "add result" and "delete" buttons.


**Table headers**

- [x] Ensure that the spacing between the headers matches Figma(I am primarily talking about the "Term" here).
- [ ] Add tracking-[5%] to the headers, this will give the letters the needed spacing.


**Table fields**

- [x] Add tracking-[0.5px].
- [ ] Change the bg color of the grey fields to match a rgb code of rgb249,249,249, and hover color to rgb 209,209,209. Probably in styles.css.
- [ ] Change color of "Edit" to --color-primary from styles.css
- [ ] Change bg color of delete button to --color-accent from styles.css. Remember to add the correct hover color as well.
- [x] Move both "Edit" and "Delete" buttons to the end of the table, maybe flex end?


### Mobile

**Sidebar menu**

- [x] Get an approved menu from your classmates. You can use the ones from develop branch: student-profile.html!

**Breadcrumbs**

- [x] Increase the font size to ensure the correct structure.
- [x] Delete the arrows.

**H1 & Co**

- [x] Ensure that everything is centered.
- [x] Switch the placement of icon and "Here are the results etc."
- [x] Make "$User" bold.
- [x] Tracking of heading should be 0.5%
- [x] Tracing of "Here are the results etc." should be 0.5px
- [ ] Get search bar from develop branch: student-profile.html.
- [x] Check that the correct color and font is used in "Add result".
- [ ] Add the plus icon.
- [x] Delete the long "Delete"-button.

**Table**

- [ ] Increase width to decrease the gap on both sides.

_Table headers_

- [ ] Add tracking 5%.
- [x] Ensure correct color.

_Table fields_

- [x] Ensure correct bg color.

**Subject pop-up**
- [ ] Make the subject pop up menu.

MarcoSaba71ni committed:
buttons, mobile breadcrumbs, bold <p>

MarcoSaba71ni commented:
Hey, Gustav.
I am having some issues and impediments that are blocking my development. If you could give me a hand with those and we can work together on it:

- [ ] On the Table field,  "change the bg color of the grey fields to match a rgb code of rgb249,249,249, and hover color to rgb 209,209,209. Probably in styles.css." isn't changing the color when hovering. I guess it is because the colors are too similar

- [ ] Not being able to change the Edit button to (--color-primary)

- [ ] When you say " Get search bar from develop branch: student-profile.html." there is not search bar on the student-profile.html

- [ ] I am not being able to find the + icon to add to the "Add Result" mobile button

- [ ] When it comes to the mobile tracking header. I don't think is making a lot of difference.

- [ ] Did not understand the "subject Pop up" on the list.

MarcoSaba71ni committed:
Edit colors, field, tracking, bg color

MarcoSaba71ni committed:
update branchs-name and content

updated branch's name (student-results => your-results). Modified content

GustavBoe reviewed (changes requested):
Hi, Marco! Great job on this, there is not a lot of things to be done. I will give you some points for you to check as you go!

### For desktop

**Head section**
- [x] Change the title of the page to your results, and the title of the file to your-results.html.
- [x] Change the description of the page to fit the content.

**Header and breadcrumbs**
- [x] Add the line on the bottom of the header (ignore if you have been told otherwise)
- [x] Move the logo to the left to ensure the correct spacing.
- [x] Change the breadcrumb from "result" to "results".

**Search bar**
- [x] Put the right placeholder in the search bar.

**Table**
- [x] Change the background color to white to get the first line the correct color.
- [x] Give the fields their correct color and text.
- [x] Move the text of the fields towards the top of the field.
- [x] Check that the gap between table and pagination is equal to 2.5rem.


### For mobile

**Other**
- [x] Delete the discarded pop-up menu on the bottom of the page.

**Heading & Co**
- [x] Delete the green arrows pointing to the heading.
- [x] Switch the placement of the book icon and the text.
- [x] Expand the search bar to be closer to 90% of the page.

**Table**
- [x] Make sure that the information of the fields and their headers are correct according to Figma, not from Top-performing-students.html.
- [x] Change the color of the fields to match Figma.

**Pagination & Co**
- [x] Ensure that the gap between table and pagination is the equivalent of 3rem.
- [x]  Ensure that the gap between pagination and button is the equivalent of 2rem.
- [x]  Ensure that the gap between button and footer-image is the equivalent of 1rem.

MarcoSaba71ni committed:
First Modifications

There are still adjustments to be done, just pushing in order to have it saved

MarcoSaba71ni commented:
Adjustments:

- Replaced px-4 of image log in the header nav in order to place the img logo more to the left
- Check that the gap between table and pagination is equal to 2.5rem:
   Added a padding bottom of 40px (pb-20)
- Ensure that the gap between table and pagination is the equivalent of 3rem.
 I used the margin botton of 10 (mb-10) which is equal to 40px

MarcoSaba71ni committed:
Header, field, pagination,

Header Spacing, field color, pagination gap
